paasta_tools/utils.py

```diff
@@ -107,6 +107,15 @@ def get_cpus(self):
         cpus = self.config_dict.get('cpus')
         return cpus if cpus else .25
 
+    def get_disk(self):
+        """Gets the  amount of disk space required from the service's configuration.
+
+        Defaults to 1024 (1G) if no value is specified in the config.
+
+        :returns: The amount of disk space specified by the config, 1024 if not specified"""
+        disk = self.config_dict.get('disk')
+        return disk if disk else 1024
```

Comment:
Can you use the default value of the `dict.get` method to provide the default value? i.e. `self.config_dict.get('disk', 1024)`